From 871e403f6799df48cfbbe08eba56d473a0c34216 Mon Sep 17 00:00:00 2001
From: q234rty <q23456yuiop@gmail.com>
Date: Thu, 17 Aug 2023 15:13:01 +0800
Subject: [PATCH] Fix build with fmt 10.1.0

I have no idea whether this is a fmt bug, and if this isn't a fmt bug how did this work before, but oh well...
---
 src/ChatPage.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ChatPage.cpp b/src/ChatPage.cpp
index 4686b0f5..06d88303 100644
--- a/src/ChatPage.cpp
+++ b/src/ChatPage.cpp
@@ -1269,7 +1269,7 @@ ChatPage::getBackupVersion()
                       nhlog::crypto()->info("Our backup key {} does not match the one "
                                             "used in the online backup {}",
                                             pubkey,
-                                            auth_data["public_key"]);
+                                            auth_data["public_key"].get<std::string>());
                       cache::client()->deleteBackupVersion();
                       return;
                   }
-- 
2.42.0

